ChatViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ChatViewController.swift
+//  Antidote
+//
+//  Created by Dmytro Vorobiov on 02/12/16.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+class ChatViewController: NSViewController {
+    fileprivate var textField: NSTextField!
+
+    override func loadView() {
+        view = NSView()
+
+        createSubviews()
+        installConstraints()
```

Comment:
Creating views programmatically is as simply as that. You create view, add subviews and setup layout.